cli/src/test/snapshots/logs.spec.ts.md

```diff
@@ -4,59 +4,6 @@ The actual snapshot is saved in `logs.spec.ts.snap`.
 
 Generated by [AVA](https://ava.li).
 
-## logs
```

Comment:
i don't understand. how can the tests pass if the md missing old tests and the new ones are missing?